src/pages/Beta/Stake/DetailModal/EarnedWidget/index.tsx

```diff
@@ -72,7 +72,7 @@ const EarnedWidget: React.FC<Props> = ({ stakingInfo }) => {
         <Box>
           <Stat
             title={t('dashboardPage.earned_weeklyIncome')}
-            stat={weeklyRewardInToken}
+            stat={numeral(weeklyRewardInToken).format('0.0000a')}
```

Comment:
do we need this change ?

Reply:
Yes, it was requested to restrict it to 3-4 digits after 0. 
Ticket link: https://app.clickup.com/t/3bugx2f

Reply:
thanks